video/out/hwdec/hwdec_cuda.c

```diff
@@ -57,12 +57,12 @@ int check_cu(const struct ra_hwdec *hw, CUresult err, const char *func)
 
 #define CHECK_CU(x) check_cu(hw, (x), #x)
 
-static const cuda_interop_init interop_inits[] = {
+static const struct cuda_interop_fn *interop_fns[] = {
```

Comment:
is there a reason this isn't just `static const struct cuda_interop_fn interop_fns[]`?

Reply:
Yeah, so I can pass a NULL pointer to break the loop.

Reply:
I see. that could be replaced a `{NULL, NULL}` entry however.